features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/tenant/tenant-conf.json

```diff
@@ -1,5 +1,29 @@
 {
   "EnableMonetization" : false,
   "IsUnlimitedTierPaid" : false,
-  "ExtensionHandlerPosition": "bottom"
+  "ExtensionHandlerPosition": "bottom",
+  "RESTAPIScopes": {
+    "Scope": [
+      {
+        "Name": "API_PUBLISHER_SCOPE",
+        "Roles": "admin"
+      },
+      {
+        "Name": "API_CREATOR_SCOPE",
+        "Roles": "admin"
+      },
+      {
+        "Name": "API_CREATOR_PUBLISHER_SCOPE",
+        "Roles": "admin"
+      },
+      {
+        "Name": "API_SUBSCRIBER_SCOPE",
+        "Roles": "Internal/subscriber"
```

Comment:
We discussed about this and decided to leave it as it is. Since Internal/subscriber role exists by default this doesn't cause any issues. And for demo purposes, we can sign-up a user from the Store and invoke the Store REST APIs by a token obtained by that user.